rpc/sigpipe.go

```diff
@@ -0,0 +1,9 @@
+// +build !darwin
+
+package rpc
+
+import "net"
+
+func DisableSigPipe(c net.Conn) error {
+	return nil
```

Comment:
Right now I am just setting this on every connection we make using the library, there is nothing in either client of KBFS that only does this for mobile. This signal is already off for all Go programs, the only reason I added this code was for some reason it is currently not off on iOS Go programs (which they are going to fix for 1.9). 

I can rename the option `DarwinDisableSigPipe` as you suggest, would that work?